Godeps/_workspace/src/github.com/openshift/openshift-sdn/ovssubnet/registry/registry.go

```diff
@@ -30,27 +30,42 @@ type EtcdSubnetRegistry struct {
 	etcdCfg *EtcdConfig
 }
 
-func newMinionEvent(action, key, value string) *api.MinionEvent {
-	min := &api.MinionEvent{}
+func newNodeEvent(action, key, value string) *api.NodeEvent {
+	nodeEvent := &api.NodeEvent{}
 	switch action {
 	case "delete", "deleted", "expired":
-		min.Type = api.Deleted
+		nodeEvent.Type = api.Deleted
 	default:
-		min.Type = api.Added
+		nodeEvent.Type = api.Added
 	}
 
 	if key != "" {
-		_, min.Minion = path.Split(key)
-		return min
+		_, nodeEvent.Node = path.Split(key)
+
+		var node map[string]interface{}
```

Comment:
where is this serialized content coming from? this method of deserializing is different than the rest of the product... I'm concerned that we'll have compatibility issues between releases with field name changes